src/@orb.yml

```diff
@@ -123,6 +123,9 @@ jobs:
       manifest:
         type: string
         description: The Cloud Foundry manifest for this environment
+      vars:
+        type: string
```

Comment:
please add a default value here

Reply:
@iynere Ah, I figured it would work the same as the `manifest.yml`? Which default would you like, empty string?

Reply:
Or `package`.